command/rootchain/server/server.go

```diff
@@ -193,7 +206,6 @@ func runRootchain(ctx context.Context, outputter command.OutputFormatter, closeC
 				},
 			},
 		},
-		AutoRemove: true,
```

Comment:
This part is opinionated but when the container fails it is useful to be able to query the logs.